src/admin/class-datalayers.php

```diff
@@ -361,6 +457,59 @@ public static function get_icons() {
 		return $icons;
 	}
 
+	/**
+	 * Check if the current screen is the correct CMB2 screen.
+	 *
+	 * @return bool
+	 */
+	public static function get_correct_cmb2_screen() {
+		// Get the current screen object.
+		$screen = get_current_screen();
+
+		// Check if we are on the edit screen of a specific custom post type.
+		if ( 'owc_ok_datalayer' === $screen->post_type && 'post' === $screen->base ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Show the datalayer metabox.
+	 *
+	 * @return bool
+	 */
+	public static function show_datalayer_metabox() {
+		$correct_cmb2_screen = self::get_correct_cmb2_screen();
+
+		if ( ! $correct_cmb2_screen ) {
```

Comment:
Ik zou de variabele overslaan en gewoon direct: `if ( ! self::get_correct_cmb2_screen() ) {` en dat dan overal waar je deze check doet. De regel moet zijn: doe je nog wat met de variabele of heb je hem meer dan eens nodig? Dan een variabele, anders gewoon direct de output gebruiken.